src/pybind/mgr/hello/module.py

```diff
@@ -0,0 +1,32 @@
+
+"""
+A hello world module
+
+See doc/mgr/hello.rst for more info.
+"""
+
+from mgr_module import MgrModule
+
+class Module(MgrModule):
```

Comment:
nit, could rename this class to Hello to emphasis the fact that any class inheriting from `MgrModule` will be considered as a mgr plugin.

Reply:
I tried to rename it to 'Hello', but it did not work; from out/mgr.x.log:
2018-04-25 10:05:11.963 7fdd3dd64700 -1 mgr load_subclass_of Module not found: 'hello'
2018-04-25 10:05:11.963 7fdd3dd64700 -1 mgr load_subclass_of Traceback (most recent call last):
  File "/home/yaarit/ceph/ceph/src/pybind/mgr/hello/__init__.py", line 1, in <module>
    from .module import Module
ImportError: cannot import name Module

2018-04-25 10:05:11.963 7fdd3dd64700 -1 mgr load Class not found in module 'hello'
2018-04-25 10:05:11.963 7fdd3dd64700 -1 mgr init Error loading module 'hello': (2) No such file or directory

What am I missing?

Reply:
ahh, you need change
```py
from .module import Module
```
to 
```py
from .module import Hello
```